apps/frontend-manage/src/components/sessions/evaluation/QuestionEvaluation.tsx

```diff
@@ -190,20 +190,35 @@ function QuestionEvaluation({
                 </UserNotification>
               )}
               {showSolution &&
-                currentInstance.questionData.options.solutionRanges && (
+                currentInstance.questionData.options.solutionRanges &&
+                (currentInstance.questionData.options.solutionRanges.length ===
```

Comment:
what if we have multiple correct solutions? would it be simple to do it as a mapping here? (check if the constraints apply to all elements in the array and then just display them all as a list)

Reply:
will be done separately once types are available (together with extended error parsing for solution ranges). Otherwise this branch would require extensive testing that everything works and with types all these extensions will be a lot easier (this is the fallback for deployment)